### DISCUSSION TITLE
other sensors coming?

### DISCUSSION THREAD

**Discussion by magnus919** [alandtse/tesla#119] [Ideas]
I'm wondering if other sensors might be exposed at some point.

I'm particularly interested in the kinds of things a parent might want to know about when letting a teenager drive their Tesla.

What's the current vehicle speed? What's the detected posted speed limit? What's the performance mode? etc.

The current vehicle speed, at least, is shown in Tesla app so I'm hoping it might be exposed by API.

This integration is very cool. I'm looking forward to seeing where it goes. Thank you!

**Comment by Olen** [comment]
Speed is availale in the device_tracker:

`state_attr('device_tracker.your_car_location_tracker', 'speed')`

I don't know if the posted speed limit is available through the API.
